script/tool/lib/src/fetch_deps_command.dart

```diff
@@ -79,22 +80,22 @@ class FetchDepsCommand extends PackageLoopingCommand {
     // `pod install` requires having the platform artifacts precached. See
     // https://github.com/flutter/flutter/blob/fb7a763c640d247d090cbb373e4b3a0459ac171b/packages/flutter_tools/bin/podhelper.rb#L47
     // https://github.com/flutter/flutter/blob/fb7a763c640d247d090cbb373e4b3a0459ac171b/packages/flutter_tools/bin/podhelper.rb#L130
-    if (getBoolArg(platformIOS)) {
-      final int exitCode = await processRunner.runAndStream(
+    final bool precacheIos = getBoolArg(platformIOS);
```

Comment:
precacheIOS

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#capitalize-identifiers-consistent-with-their-spelling